### PR TITLE
feat(ui): Card component

### PR DESCRIPTION
**Pull request by lucasfp13**
## What's the purpose of this pull request?

This PR intends to add the Card organism.

## How it works? 

The `Card` component works together with its own wrapper components for each content's details:

- `CardImage`: the wrapper component for the content's image;
- `CardContent`: the wrapper component for the content's details;
- `CardActions`: the wrapper component for the content's CTAs (call-to-action).

## How to test it?

- Visually, through Storybook;
- Unit tests through Jest: yarn test packages/ui/src/organisms/Card/Card.test.tsx

**Storybook preview**:

Screen Shot 2021-12-13 at 15 40 29

## References

- [ARIA Roles](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles)
- ["How to build accessible cards–block links"](https://www.nomensa.com/blog/how-build-accessible-cards-block-links)
- [Radix UI](https://www.radix-ui.com/docs/primitives/components/)
- [Reach UI - Menu Button](https://reach.tech/menu-button/)
- [MUI Card component](https://mui.com/components/cards/)

### CODE REVIEW

**Review by tlgimenes — APPROVED** [review]
Nice! After fixing the linting messages, LGTM

File: themes/theme-b2c-tailwind/src/organisms/card.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+[data-store-card-content] [data-store-price] {
+  @apply mr-1 text-gray-500;
+}
+
+[data-store-card-content] [data-store-badge] {
+  @apply mt-2 text-xs rounded-full text-gray-800 bg-green-100;
+}
+
+[data-store-card-actions] [data-store-button] {
+  @apply mr-2 mt-2 text-sm float-right;
```
Comment (eduardoformiga) [inline themes/theme-b2c-tailwind/src/organisms/card.css:18]: The `float-right` looks different to me. But it's not so important ;)

**Review by eduardoformiga — APPROVED** [review]
Good Job 👏

**Review by saranicoly — APPROVED** [review]
Great job!

**Pull request merged**